Makefile

```diff
@@ -62,4 +63,8 @@ push-staging:
 	make APP_ENV=staging build
 	docker-compose -f docker-compose.yml -f config/staging.yml up -d
 
-.PHONY: default build serve clean debug develop develop-rebuild dropdb test
+production: APP_ENV=production
+production: DOCKER_COMPOSE_FILE=docker-compose-production.yml
+production: serve-d
```

Comment:
Sorry I commented on the wrong line. What I meant was the overriding of the `DOCKER_COMPOSE_FILE`. We already have splitting of that based on the target definition adding an extra docker file for production makes this not so clean.